integration-tests/multipart-request/src/main/openapi/multipart-requests.yml

```diff
@@ -62,7 +62,7 @@ components:
           example: New Delhi
     UserId:
       type: string
-      format: uuid
+      format: string
```

Comment:
What's the problem with this format?

Reply:
Quarkus still doesn't support it. It will be supported in the next version (2.16.4).
I can add a converter if you prefer, as I did for OffsetDateTime in https://github.com/quarkiverse/quarkus-openapi-generator/pull/262/files#diff-61172ef4aaa50da592ec53387fd6765f66e2e9c8f58feba1539a7a742112b447.

https://github.com/quarkusio/quarkus/issues/31298

Reply:
The quarkus issue: https://github.com/quarkusio/quarkus/issues/31298

Reply:
Thank u!